frontend/src/scenes/notebooks/NotebookScene.tsx

```diff
@@ -76,6 +87,13 @@ export function NotebookScene(): JSX.Element {
                         items={[
                             {
                                 items: [
+                                    sharingIsAllowed &&
+                                        !notebook?.is_template && {
+                                            label: 'Share',
+                                            onClick: () => {
+                                                push(urls.notebookShare(notebookId))
```

Comment:
I'm guessing we use the `push` method here to avoid actually setting the notebook into `NotebookMode.Share` via the `setNotebookMode` method as is done for View / Edit toggles:
https://github.com/PostHog/posthog/blob/04c54ddb2cf43b5d539aeda4a631c259de13d013/frontend/src/scenes/notebooks/NotebookScene.tsx#L131-L143

Reply:
Using the URL to open the modal is only to keep the mechanism the same between the different types of shareable thing. With the added bonus that it's nice for states to be addressable -> "You can click this link if you want to stop sharing that notebook"